### DISCUSSION TITLE
Как скачать песни из "Моя волна"?

### DISCUSSION THREAD

**Discussion by Crd5** [MarshalX/yandex-music-api#609] [Q&A] [answered]

**Comment by MarshalX** [comment, marked as answer]
"Моя волна" это радиостанция с ID user:onyourwave. Получаем треки радиостанции (см. пример радио) и полученный трек скачиваем по ID (см. ридми с примером скачивания трека)

**Comment by Crd5** [comment]
Можете привести пример пожалуйста @MarshalX

**Reply by ensiouel** [reply]
Приложу свое решение по скачиванию "Моя волна" с Яндекс.Музыки, возможно, сэкономит вам время: https://gist.github.com/ensiouel/f5e88e90d2c95ae370590d1402970b39

Спасибо огромное авторам библиотеки
